src/sql/src/session/vars.rs

```diff
@@ -1404,6 +1408,21 @@ impl SystemVars {
         Self::SESSION_VARS.contains_key(UncasedStr::new(name)) || name == ENABLE_RBAC_CHECKS.name()
     }
 
+    /// Returns a [`SystemVar`] representing the configuration parameter with the
+    /// specified name.
+    ///
+    /// See [`Self::get`] for more details.
+    ///
+    /// # Errors
+    ///
+    /// The call will return an error:
+    /// 1. If `name` does not refer to a valid [`SystemVars`] field.
+    fn get_sys_var(&self, name: &str) -> Result<&SystemVar, VarError> {
+        self.vars
+            .get(UncasedStr::new(name))
+            .ok_or_else(|| VarError::UnknownParameter(name.into()))
+    }
```

Comment:
It looks like this is only called from `SystemVars::get(...)`? What do you think about merging these two functions back together?

Reply:
Yeah that looks way better. I'm not sure why I split them, I think I had a different approach at some point.